data/wsgi.ini.template

```diff
@@ -1,7 +1,7 @@
 [wsgi]
 workdir = workdir
-reference_housenumbers = refdir/hazszamok_20220124.tsv refdir/hazszamok_kieg_20220123.tsv
-reference_street = refdir/utcak_20220123.tsv
+reference_housenumbers = refdir/hazszamok_20220125.tsv refdir/hazszamok_kieg_20220125.tsv
+reference_street = refdir/utcak_20220125.tsv
 reference_citycounts = refdir/varosok_count_20220123.tsv
```

Comment:
majd a következőnél a count-okat is frissíteni kell

Reply:
Beraktam: https://github.com/vmiklos/osm-gimmisn/commit/be4801411ffc9f56bdb25ff2006c2148ef4ef254